src/createServer.js

```diff
@@ -1,3 +1,40 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+/* eslint-disable max-len */
```

Comment:
Avoid disabling eslint rules without a valid reason. It is better to adjust the code to meet the eslint rules.